### DISCUSSION TITLE
Motion Tracking

### DISCUSSION THREAD

**Discussion by Atlas-Night-Out** [NightscoutFoundation/xDrip#2802] [Ideas]
Keeping my mouth shut!

**Comment by Navid200** [comment]
Can you provide more detail?
What would be a simple explanation?

xDrip emergency sends a message to someone if you don't snooze your alert.  This could be because you are unconsious, whether you have fallen or not.
Why is that no adequate?

**Reply by Navid200** [reply]
We have to decide what xDrip is and what it isn't.  We cannot keep adding features to it.
We add features that are necessary.  Otherwise, if we add every single request with no questions asked, it will be impossible to maintain the app.   A lot of work goes into keeping this app going that you may not be aware of.
And it will become extremely difficult to use as it will become more and more complicated.  
My understanding is that fall detect apps already exist. 

If a request can be accomplished with the app as is, in other words, if the app is adequate, there is no reason to add it.

Also, xDrip is not a medical app.  It is a research tool.  Anything you use it for, is your personal decision.
We don't advise anyone to use xDrip for anything related to any medical condition. or scenario
